model/migrations/28ce26f-conversation_state.sql

```diff
@@ -0,0 +1,13 @@
+--
+-- Table structure for table `conversation_state`
+--
+DROP TABLE IF EXISTS `conversation_state`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `conversation_state` {
+  `conversationId` varchar(255) COLLATE utf8mb4_bin not NULL,
```

Comment:
One thing that Antonio noted we need to track in the conversation state is also the last message ID, otherwise message IDs reset when the engine is restarted and that confuses clients.

Reply:
Makes sense, just added it along with the go code.